pkgs/misc/anarchism/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "anarchism-${version}";
```

Comment:
`anarchism` is not a proper name for this work. It should be `afaq`, or `an-anarchist-faq`.

Reply:
It has been for many years in debian world, on archlinux and whatnot. It's a well known package and often refered to by it's pkg name and I'd like to keep it that way.

Reply:
Context: https://lists.debian.org/debian-devel/1999/03/msg01628.html

Including this package in Debian has arguably been a mistake that I wish will not be repeated here. Software distributions are not the proper place for political debate.